demos/classification_demo/cpp/main.cpp

```diff
@@ -324,7 +324,7 @@ int main(int argc, char *argv[]) {
             }
         }
 
-        //// --------------------------- Report metrics -------------------------------------------------------
+        // --------------------------- Report metrics -------------------------------------------------------
```

Comment:
Since we have a log message below, such comments become redundant.

Reply:
Removed in all demos.